notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>I'm confused about what you're doing here. In https://cloud.devsite.corp.google.com/vertex-ai/docs/tabular-data/tabular-workflows/forecasting-train?db=reznitskii#previous-result, we query "artifacts[0].uri" and use it to set the variable stage_1_tuning_result_artifact_uri. Here, you use metadata["resourceName"] to create a forecasting_mp_model, which is then not used for the skip architecture search pipeline?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Sorry for the confusion. You are right, this section is specifically for retrieving the tuning result. The upload model part is to retrieve the Vertex model resource which will be used for batch prediction later, I have moved below to the batch prediction section.</p>